index.js

```diff
@@ -54,6 +54,7 @@ class Analytics {
     this.flushAt = Math.max(options.flushAt, 1) || 20
     this.flushInterval = options.flushInterval || 10000
     this.flushed = false
+    this.enabled = options.enable === false ? false : true // eslint-disable-line no-unneeded-ternary
```

Comment:
Strange use of a ternary here (seems the linter agrees with me). Any reason `this.enabled = options.enable` won't work?

Reply:
No idea what happened here 😆